x-pack/plugins/synthetics/server/routes/settings/enable_default_alerting.ts

```
@@ -0,0 +1,160 @@
+    settings?.defaultConnectors?.includes(act.id)
+  );
+
+  const actions: RuleAction[] = populateAlertActions({
+    groupId: ACTION_GROUP_DEFINITIONS.MONITOR_STATUS.id,
+    defaultActions,
+    defaultEmail: settings?.defaultEmail!,
+    translations: {
+      defaultActionMessage: SyntheticsMonitorStatusTranslations.defaultActionMessage,
+      defaultRecoveryMessage: SyntheticsMonitorStatusTranslations.defaultRecoveryMessage,
+      defaultSubjectMessage: SyntheticsMonitorStatusTranslations.defaultSubjectMessage,
+    },
+  });
+
+  if (data.length === 0) {
+    const alert = await rulesClient.create<{}>({
+      data: {
+        actions,
+        params: {},
+        consumer: 'uptime',
+        alertTypeId: SYNTHETICS_ALERT_RULE_TYPES.MONITOR_STATUS,
+        schedule: { interval: '1m' },
+        notifyWhen: 'onActionGroupChange',
+        tags: ['SYNTHETICS_DEFAULT_ALERT'],
+        name: `Synthetics internal alert`,
+        enabled: true,
+        throttle: null,
+      },
+    });
+    return { ...alert, ruleTypeId: alert.alertTypeId };
+  } else {
+    const alert = data[0];
+    const updatedAlert = await rulesClient.update({
+      id: alert.id,
+      data: {
+        actions,
+        name: alert.name,
+        tags: alert.tags,
+        schedule: alert.schedule,
+        params: alert.params,
+        notifyWhen: alert.notifyWhen,
+      },
+    });
+    return { ...updatedAlert, ruleTypeId: updatedAlert.alertTypeId };
+  }
+};
+
+const getActionConnectors = async (
+  context: UptimeRequestHandlerContext,
+  soClient: SavedObjectsClientContract,
+  server: UptimeServerSetup
+) => {
+  const actionsClient = (await context.actions)?.getActionsClient();
+
+  const settings = await savedObjectsAdapter.getUptimeDynamicSettings(soClient);
+  let actionConnectors: FindActionResult[] = [];
+  try {
+    actionConnectors = await actionsClient.getAll();
+  } catch (e) {
+    server.logger.error(e);
```

We may still need to throw this error to support any kind of error handling UX.